src/test/resources/fixtures/hmh/hmh-interacted/development/caliperEventAssessementStarted.json

```diff
@@ -0,0 +1,86 @@
+                "type": "AssessmentEvent",
+                "actor": "https://hmhco.com/ims/oneroster/v1p1/users/ff7c53fd-9024-4d2d-8503-aa3da2cbc73e",
+                "action": "Started",
+                "object": {
+                    "id": "data-uuid-e2a6e58e709f4b15b1246dcbe88c6068",
+                    "type": "Assessment",
+                    "name": "CPL v2 Short text table",
+                    "description": "This is the widget CPL Short text table",
+                    "dateCreated": "2019-08-29T15:11:04.271Z",
+                    "dateModified": "2019-08-29T15:11:04.271Z",
+                    "datePublished": "2019-08-29T15:11:04.271Z",
+                    "dateToActivate": "2019-08-29T15:11:04.271Z",
+                    "dateToShow": "2019-08-29T15:11:04.271Z",
+                    "dateToStartOn": "2019-08-29T15:11:04.271Z",
+                    "dateToSubmit": "2019-08-29T15:11:04.271Z",
+                    "maxAttempts": 0,
+                    "extensions": {
+                        "contentRootId": "080c58111f020b9418f824970ffd7bb5392",
+                        "contentRootType": "pubId",
+                        "contentVersionId": 0,
+                        "userRole": "student",
+                        "contentItemId": "e48e0c50555249d8adec72c566953b2d"
+                    }
+                },
+                "eventTime": "2019-08-29T14:11:04.283Z",
+                "target": {
+                    "id": "http://rce.localhost/content/book/OPS/s9ml/cards/el_uat_content_samp_019.xhtml",
+                    "type": "Frame",
+                    "name": "Short Text Table",
+                    "extensions": {
+                        "contentRootId": "080c58111f020b9418f824970ffd7bb5392",
+                        "contentRootType": "pubId",
+                        "contentVersionId": 0,
+                        "userRole": "student",
+                        "contentItemId": "e48e0c50555249d8adec72c566953b2d"
+                    },
+                    "isPartOf": {
+                        "id": "http://rce.localhost/content/book/OPS/s9ml/cards/el_uat_content_samp_001.xhtml",
+                        "type": "DigitalResource",
+                        "name": "Module 1: QA Review",
+                        "isPartOf": {
+                            "id": "http://rce.localhost/digitalresource/080c58111f020b9418f824970ffd7bb5392",
+                            "type": "DigitalResource",
+                            "name": "Experience Library Interaction QA Review",
+                            "isPartOf": {
+                                "id": "NoProgrammeId",
+                                "type": "DigitalResource",
+                                "name": "NoProgramme"
+                            }
+                        }
+                    }
+                },
+                "edApp": {
+                    "id": "http://rce.localhost/rce",
```

Comment:
Should the ID be RCE for the HMH-Interacted Event?

Reply:
It's still event ed from within RCE with the book/page context so I think it should still 'live' in the RCE edApp space. It doesn't work out side of that context